keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -263,17 +482,166 @@ When downgrading from a release with this feature, to a release without maxUnava
 - maxUnavailable greater than 1 with partition and staged pods greater than maxUnavailable
 - maxUnavailable greater than 1 with partition and maxUnavailable greater than replicas
 
-## Test Plan
-For `Alpha`, unit tests and e2e tests will be added to test functionality at both
-with feature flag enabled and disabled. Defaults will be verified so that users
-who donot set this flag are not surprised at all.
+Coverage:
+
+- `pkg/apis/apps/v1`:               `2023-05-26` - `71.7%`
+- `pkg/apis/apps/v1beta2`:          `2023-05-26` - `76.7%`
+- `pkg/apis/apps/validation`:       `2023-05-26` - `92.3%`
+- `pkg/controller/statefulset`:     `2023-05-26` - `85.7%`
+- `pkg/registry/apps/statefulset`:  `2023-05-26` - `63.9%`
```

Comment:
Is there a coverage goal? I'm not sure what to make of this.

Reply:
No, just enumerate the core package in the form as suggested.